abis/linux/in.h

```diff
@@ -44,6 +44,8 @@ struct ipv6_mreq {
 	unsigned        ipv6mr_interface;
 };
 
+extern const struct in6_addr in6addr_any;
```

Comment:
this isn't defined afaict?

Reply:
Hmm, likely that this isn't needed at all as this line is already declared in `options/posix/include/netinet/in.h`. The actual definition is in `options/posix/generic/arpa-inet-stubs.cpp`. This should probably be dropped.